src/config/experiment.js

```diff
@@ -1,22 +1,64 @@
-import { defaultBlockSettings } from './main';
-import { deepCopy } from '../lib/utils'
+import { deepCopy } from "../lib/utils"
+import localConfig from "./config.json"
+import { envConfig } from "./main"
+import path from "path"
+import { firestoreConfig } from "../firebase"
```

Comment:
should this function be called `getFirestoreConfig`? along side the other imports it seems like this would be some sort of config object, which isn't the case